.github/CODEOWNERS

```diff
@@ -1,86 +1,35 @@
 * @vladimirlaz @romanovvlad @bader
 
-# AOT compilation
-SYCL/AOT @AGindinson @dm-vodopyanov @AlexeySachkov @romanovvlad
+# Use runtime team as the umbrella for most of the tests
+SYCL @intel/llvm-reviewers-runtime
 
-# Atomics
-SYCL/AtomicRef @AGindinson 
+# AOT compilation
+SYCL/AOT @intel/dpcpp-tools-reviewers
 
-# SYCL RT specific tests
-SYCL/Assert @intel/llvm-reviewers-runtime
-SYCL/Basic @intel/llvm-reviewers-runtime
-SYCL/Config @intel/llvm-reviewers-runtime
-SYCL/DiscardEvents @intel/llvm-reviewers-runtime
-SYCL/FilterSelector @intel/llvm-reviewers-runtime
-SYCL/HostInteropTask @intel/llvm-reviewers-runtime
-SYCL/InorderQueue @intel/llvm-reviewers-runtime
-SYCL/KernelAndProgram @intel/llvm-reviewers-runtime
-SYCL/KernelParams @intel/llvm-reviewers-runtime
-SYCL/Scheduler @intel/llvm-reviewers-runtime
-SYCL/Tracing  @intel/llvm-reviewers-runtime
+#BFloat16 conversion
+SYCL/BFloat16 @AlexeySotkin @intel/dpcpp-tools-reviewers
 
 # Device code split
-SYCL/DeviceCodeSplit @AlexeySachkov  @Fznamznon
-
-# Device library
-SYCL/DeviceLib @vzakhari
-SYCL/DeviceLib/ITTAnnotations @vzakhari @MrSidims @AGindinson
-
-# dot_product API
-SYCL/DotProduct @rdeodhar
+SYCL/DeviceCodeSplit @intel/dpcpp-tools-reviewers
 
 # Explicit SIMD
-SYCL/ESIMD @kbobrovs @v-klochkov @sndmitriev
-
-# Functor
-SYCL/Functor @AlexeySachkov
+SYCL/ESIMD @intel/dpcpp-esimd-reviewers
 
 # Group algorithms
-SYCL/GroupAlgorithm @Pennycook @AlexeySachkov
-SYCL/SubGroup @Pennycook @AlexeySachkov
-SYCL/SubGroupMask @Pennycook @vladimilaz
-
-# Group local memory
-SYCL/GroupLocalMemory @sergey-semenov @Pennycook
-
-# Hierarchical parallelism
-SYCL/HierPar @kbobrovs
-
-# Inline assembler
-SYCL/InlineAsm @AlexeySachkov
+SYCL/GroupAlgorithm @Pennycook @intel/llvm-reviewers-runtime
+SYCL/SubGroup @Pennycook @intel/llvm-reviewers-runtime
+SYCL/SubGroupMask @Pennycook @intel/llvm-reviewers-runtime
+SYCL/GroupLocalMemory @Pennycook @intel/llvm-reviewers-runtime
 
-# Online compiler
-SYCL/OnlineCompiler @v-klochkov
-
-# Plugin interface
-SYCL/Plugin @smaslov-intel
+# Plugin interface for Level Zero
+SYCL/Plugin/*level-zero* @intel/dpcpp-l0-pi-reviewers
+SYCL/Plugin/*level_zero* @intel/dpcpp-l0-pi-reviewers
```

Comment:
This looks like a copy-paste bug.

Reply:
we have tests with a minus in name and underline sign. It is expected

Reply:
What is the reason of this inconsistency?